Scenes/Screens/RevivalShop/RevivalShopItem.gd

```diff
@@ -2,7 +2,25 @@ extends HBoxContainer
 
 export (Color, RGB) var color
 export (String) var info
+export (String) var return_func
+export (String) var upgrade_func
+export (String) var button_sign
+var count_text
+
 
 func _ready():
 	$Info.set_text(info)
 	$Count.set("custom_colors/font_color", color)
+	
+func update():
```

Comment:
`update` method is already in Godot: [READ](https://docs.godotengine.org/en/3.2/tutorials/2d/custom_drawing_in_2d.html?#updating). Maybe `refresh`? `_update`? `_update_self`? I don't have got 'nice' idea now... But surely NOT just `update`.